AElf.Kernel/AccountManager.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AElf.Kernel
+{
+    class AccountManager : IAccountManager
+    {
+        public List<IAccount> ExecuteTransactionAsync(ITransaction tx)
+        {
+            var accountFrom = tx.AccountFrom;
+            var accountTo = tx.AccountFrom;
+            var amount = tx.Amount;
+
+            var stateFrom = WorldState.GetAmountByAccount(accountFrom);
+            var stateTo = WorldState.GetAmountByAccount(accountTo);
```

Comment:
the world state should be changed by 
var invoker = Account.GetInvoker(Caller,Transaction);//Invoker here seems like a gateway to the Virtual Machine/Service
var worldStateChanges = await invoker.InvokeAsync(WorldState);

worldStateManager.MergeWorldStateAsync(WorldState, worldStateChanges);